hw/ip/otbn/data/enc-schemes.yml

```diff
@@ -309,27 +315,43 @@ bnxid:
     - custom0
     - funct3
   fields:
-    imm: 24-22,31-25
-    spp: 21
-    dpp: 20
-    rs: 19-15
-    rd: 11-7
+    imm: 11-9,31-25
+    rs2: 24-20
+    rs1: 19-15
+    spp: 8
+    dpp: 7
 
-# Used by bn.mov and bn.movr
+# Used by bn.mov
 bnmov:
   parents:
     - custom0
     - funct3(funct3=b110)
+    - indirect(indirect=b0)
   fields:
-    indirect: 31
     fixed_top:
-      bits: 30-22
-      value: bxxxxxxxxx
-    spp: 21
-    dpp: 20
+      bits: 30-20
+      value: bxxxxxxxxxxx
     src: 19-15
     dst: 11-7
 
+# Used by bn.movr
+bnmovr:
+  parents:
+    - custom0
+    - funct3(funct3=b110)
+    - indirect(indirect=b1)
+  fields:
+    fixed_top:
+      bits: 30-25
+      value: bxxxxxx
+    dst: 24-20
+    src: 19-15
+    spp: 9
+    dpp: 7
```

Comment:
The subtle difference with `BN.LID` `BN.SID` here (where spp is at 8) is because for `BN.LID`, `BN.SID` it represents a WLEN sized increment, here it's a simple +1 increment. I could rename things so they don't both use 'spp' here.

Reply:
I don't think the name really matters: when I designed this YAML stuff, the field names were just to allow instructions to talk about regions of the encoding; there's not necessarily any meaning to the names between different encoding schemes.